pkgs/development/libraries/snappy/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchFromGitHub, cmake }:
+{ stdenv, fetchFromGitHub, cmake, static ? false }:
```

Comment:
`enableShared`?

Reply:
Ah, this is exclusive shared or static, I guess.

Reply:
See below. `enableShared` would suggest that shared and static libs would both appear in the output, but that is not possible.

Reply:
@tobim Yes, thank you.